.changeset/mighty-forks-exercise.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/plugin-catalog-backend': patch
+---
+
+Changed the target column in the locations table to TEXT type.
+Added a hash for the key column in the `refresh_keys` table.
```

Comment:
@freben afaik this was `string` to support MySQL, see for example https://github.com/backstage/backstage/blob/cfc3ca6ce060315b461a96a0aababf2a6590c207/plugins/catalog-backend/migrations/20210302150147_refresh_state.js#L93-L95

Do you know if that's no longer a requirement, or never was?

Reply:
I believe that's what it needs to be to be able to be indexed. If it's not indexed (and preferably also covered by migrations tests), it should be ok

Reply:
yeah I double checked, only the id column is indexed on that table